src/editor-service/editor-server.ts

```diff
@@ -97,7 +97,7 @@ process.stdin.pipe(split()).on('data', async function(line: string) {
 // node child_process.fork() IPC interface
 process.on('message', async function(request: RequestWrapper) {
   const result = await getSettledValue(request.value);
-  process.send(<ResponseWrapper>{id: request.id, value: result});
+  process.send!(<ResponseWrapper>{id: request.id, value: result});
```

Comment:
It means "I solemnly swear the expression to the left of the `!` is never null or undefined (right here just this one time)"